MAINTENANCE.md

```diff
@@ -0,0 +1,50 @@
+# Migrate AWS RDS databases to use different private subnets
+
+This procedure explains how to migrate AWS RDS databases, deployed with reference architecture, from using separate, private DB subnets to (re-)using
+private subnets already deployed with VPC. This only applies to already existing deployments, new deployments are not affected.
+The change was introduced with v0.2.0 of reference architecture.
```

Comment:
The change was  introduced with v0.2.0 of Simphera AWS reference architecture.

Reply:
Done